src/import/Import.ml

```diff
@@ -18,8 +18,8 @@ include (
 
       include
         module type of Base
-        (* [Filename], [Format], [Scanf] are all deprecated in [Base], erase
-           them and use the ones from the stdlib. *)
+          (* [Filename], [Format], [Scanf] are all deprecated in [Base],
+             erase them and use the ones from the stdlib. *)
```

Comment:
The new indentation looks more natural to me with the new boxing

Reply:
I'm not sure I agree, but it's not a big deal, and the rest of the effect of this patch is a significant improvement.